HLTrigger/Configuration/python/HLT_75e33/modules/hltPuppiTauTkIsoEle4522L1TkFilter_cfi.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+hltPuppiTauTkIsoEle4522L1TkFilter = cms.EDFilter("PathStatusFilter",
```

Comment:
I'd suggest to change the name of the module and of the files by adding an underscore between 45 and 22.

Reply:
Done.

Reply:
@rdewanje now this file is completely superseded by `HLTrigger/Configuration/python/HLT_75e33/modules/hltPuppiTauTkIsoEle45_22L1TkFilter_cfi.py`. 
Please delete it and squash the commits in order to avoid having "virtual" files in the git history.

Reply:
Done.